### DISCUSSION TITLE
《绝区零》青衣MV丨红透晚烟青

### DISCUSSION THREAD

**Discussion by justafish8787** [share121/inter-knot#1833] [其他]
![image](https://github.com/user-attachments/assets/858e323e-7fce-4e46-a781-79a13054a13c)
万万遍地重置，记忆早已灭失。
缘自何时起？又往何处去？
世界待我狰狞，
这一世，
是否是我的依凭？

▼ 制作名单
制作人：杨武韬
演唱：金玟岐
戏剧念白：裘丹莉
作词：陈如业
作曲/编曲：杨武韬
录音师/人声编辑：徐威
混音/母带：黄钦胜
管弦乐/民乐：国际首席爱乐乐团
录音棚：52Hz Studio

[点击此处观看]( https://www.bilibili.com/video/BV11S421X7hy)

**Comment by gaojiaobei** [comment]
青衣！豪堪🥰

**Comment by ZhuYuan-148** [comment]
前辈好帅！
